niceman/interface/diff.py

```diff
@@ -62,17 +69,22 @@ def __call__(prov1, prov2):
         env_1 = NicemanProvenance(prov1).get_environment()
         env_2 = NicemanProvenance(prov2).get_environment()
 
-        for (dist_type, pkg_type) in ((DebianDistribution, "Debian package"), 
-                                      (CondaDistribution, "Conda package")):
+        # pkg here could be a package or a repository
+        for (dist_type, pkg_type) in (
+                (DebianDistribution, "Debian package"),
+                (CondaDistribution, "Conda package"),
+                (GitDistribution, 'Git repository'),
+                (SVNDistribution, 'SVN repository')
+        ):
 
             dist_1 = env_1.get_distribution(dist_type)
             if dist_1:
-                pkgs_1 = { p._cmp_id: p for p in dist_1.packages }
+                pkgs_1 = {p._cmp_id: p for p in dist_1.packages}
```

Comment:
It'd be better if you'd do purely cosmetic changes like this and the one below in a separate commit.